src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+// #region card
+
+.card {
+  background-color: #fff;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    margin-top: 32px;
+  }
+
+  &__text-conteiner {
```

Comment:
There is a typo in the class name `&__text-conteiner`. It should be `&__text-container` to match the corrected HTML class name.